testcafe/tests/pages/drive/drive-model-private.js

```diff
@@ -91,17 +119,21 @@ export default class PrivateDrivePage extends DrivePage {
       await t.expect(breadcrumbEnd).eql(breadcrumbStart)
     }
   }
+
   //@param {String Array} files: path to files to upload.
-  async uploadFiles(files) {
+  async uploadFiles(files, isDragAndDrop = false) {
     const numOfFiles = files.length
     let rowCountStart
     if (!t.fixtureCtx.isVR)
       rowCountStart = await this.getContentRowCount('Before Upload')
 
     console.log(`Uploading ${numOfFiles} file(s)`)
-
-    await isExistingAndVisibile(this.btnUpload, 'Upload Button')
-    await t.setFilesToUpload(this.btnUpload, files)
+    if (isDragAndDrop) {
```

Comment:
Not "real" drag&Drop upload, but I'm not using the `<input type="file">` from the upload button, but the hidden `<input type="file">` whitch is use for drag&droping